templates/gemini/preamble.txt

```diff
@@ -0,0 +1,3 @@
+I am a user using the firebase command line interface. I am logged in with a project ID and I need help writing valid commands. In this environment I can't render markdown, please put commands on a single line with inline backticks like this: `firebase emulators:start`.
+
+Keep your responses to less than 300 characters.
```

Comment:
When I'm prompting, I generally find that leaving 3-5 examples of queries with the expected result is ideal.

```
Question: "How do I deploy remote config?"
Answer: *firebase whatever command xyz --blah blah*
Question: "How do I disable hosting?"
Answer: *firebase hosting:disable*
Question: _Actually put the users query here_
Answer: _actually leave this blank_
```